src/App.js

```diff
@@ -1,46 +1,29 @@
 import React from 'react';
 import './App.scss';
-
-const misha = {
-  name: 'Misha',
-  age: 37,
-  sex: 'm',
-  isMarried: true,
-  partnerName: 'Natasha',
-};
-
-const olya = {
-  name: 'Olya',
-  sex: 'f',
-  isMarried: true,
-  partnerName: 'Maksym',
-};
-
-const alex = {
-  name: 'Alex',
-  age: 25,
-  sex: 'm',
-  isMarried: false,
-};
```

Comment:
Don't remove these objects. Pass them as props to your `Person` component:
`<Person { ... misha }`